src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -262,9 +264,13 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
    * @param is   the input stream to write from
    * @return `true` if a new file was created, `false` otherwise
    */
-  def writeDepositFile(is: => InputStream, user: String, id: UUID, path: Path): Try[Boolean] = {
+  def writeDepositFile(is: => InputStream, user: String, id: UUID, path: Path, contentType: String): Try[Boolean] = {
     for {
+      _ <- if (Option(contentType).toSeq.exists(!_.trim.matches(contentTypeZipPattern))) Success(())
```

Comment:
I don't think `.toSeq` is necessary here. `exists` is also defined on `Option`

```suggestion
      _ <- if (Option(contentType).exists(!_.trim.matches(contentTypeZipPattern))) Success(())
```